es/cheatsheet-supervised-learning.md

```diff
@@ -1,567 +1,566 @@
 
-&#10230;
+&#10230; Error de entrenamiento - Para un clasificador dado h, se define el error de entrenamiento ˆϵ(h), también conocido como riesgo empírico o error empírico, de la siguiente forma:
 
 <br>
 
 **80. Probably Approximately Correct (PAC) ― PAC is a framework under which numerous results on learning theory were proved, and has the following set of assumptions: **
 
-&#10230;
+&#10230; Aprendizaje correcto probablemente aproximado (en inglés, probably approximately correct learning) (PAC) - PAC es un marco bajo el cual se probaron numerosos resultados en teoría de aprendizaje, y presenta los siguientes supuestos:
 
 <br>
 
 **81: the training and testing sets follow the same distribution **
 
-&#10230;
+&#10230; los conjuntos de entrenamiento y de prueba siguen la misma distribución
 
 <br>
 
 **82. the training examples are drawn independently**
 
-&#10230;
+&#10230; los ejemplos de entrenamiento son escogidos de forma independiente
 
 <br>
 
 **83. Shattering ― Given a set S={x(1),...,x(d)}, and a set of classifiers H, we say that H shatters S if for any set of labels {y(1),...,y(d)}, we have:**
 
-&#10230;
+&#10230; Shattering - Dado un conjunto S={x(1),...,x(d)}, y un conjunto de clasificadores H, decimos que H destroza (shatters) S si para cualquier conjunto de etiquietas {y(1),...,y(d)}, tenemos que:
 
 <br>
 
 **84. Upper bound theorem ― Let H be a finite hypothesis class such that |H|=k and let δ and the sample size m be fixed. Then, with probability of at least 1−δ, we have:**
 
-&#10230;
+&#10230; Teorema de la frontera superior - Sea H una clase de hipótesis finita tal que |H|=k y sea δ y el tamaño de la muestra m fijo. Entonces, con probabilidad de al menos 1−δ, tenemos:
 
 <br>
 
 **85. VC dimension ― The Vapnik-Chervonenkis (VC) dimension of a given infinite hypothesis class H, noted VC(H) is the size of the largest set that is shattered by H.**
 
-&#10230;
+&#10230; Dimensión VC - La dimensión de Vapnik-Chervonenkis (VC) de una clase de hipótesis finita H, denotada como VC(H), es el tamaño del conjunto más grande destrozado (shattered) por H.
 
 <br>
 
 **86. Remark: the VC dimension of H={set of linear classifiers in 2 dimensions} is 3.**
 
-&#10230;
+&#10230; Observación: la dimensión VC de H={conjunto de clasificadores lineales en dos dimensiones} es 3.
 
 <br>
 
 **87. Theorem (Vapnik) ― Let H be given, with VC(H)=d and m the number of training examples. With probability at least 1−δ, we have:**
 
-&#10230;
+&#10230; Teorema (Vapnik) - Dado H, con VC(H)=d y m el número de ejemplos de entrenamiento. Con probabilidad de al menos 1−δ, tenemos que:
 
 <br>
 
 **88. [Introduction, Type of prediction, Type of model]**
 
-&#10230;
+&#10230; [Introducción, Tipo de predicción, Tipo de modelo]
 
 <br>
 
 **89. [Notations and general concepts, loss function, gradient descent, likelihood]**
 
-&#10230;
+&#10230; [Notaciones y conceptos generales, función de perdida, descenso del gradiente, probabilidad]
```

Comment:
Again, consider to use "[_Descenso de gradiente_](https://developers.google.com/machine-learning/glossary/#gradient_descent)".